src/components/RegisterPage/RegisterPage.js

```diff
@@ -1,9 +1,169 @@
 import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Form, Field } from 'react-final-form';
+import {
+  Container,
+  Row,
+  Col,
+  Button,
+  FormGroup,
+  Label,
+  Input,
+  Card,
+  CardBody,
+} from 'reactstrap';
 
-const RegisterPage = () => (
-  <h2>
-    Register Page
-  </h2>
-);
+
+import { routes } from '../../constants';
+import { history } from '../../helpers';
+import { SESSION_STORAGE_KEY } from '../../constants/auth.constant';
+import { validations } from '../../helpers/validations';
+
+import FormError from '../FormError';
+
+class RegisterPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    if (sessionStorage.getItem(SESSION_STORAGE_KEY)) {
+      history.push('/');
```

Comment:
Use home key from routes.js
The root '/' isn't gonna change, still it's a good practice of using exported strings instead of writing raw.